scripts/age-onset-json.py

```diff
@@ -0,0 +1,147 @@
+                if inheritance in plot_data:
+                    plot_data[inheritance]["Onset"]["y"].append(disease)
+                    plot_data[inheritance]["Onset"]["base"].append(onset_min)
+                    plot_data[inheritance]["Onset"]["x"].append(onset_max-onset_min)
+                    plot_data[inheritance]["TypOnset"]["y"].append(disease)
+                    plot_data[inheritance]["TypOnset"]["base"].append(typonset_min)
+                    plot_data[inheritance]["TypOnset"]["x"].append(typonset_max-typonset_min)
+                else:
+                    plot_data[inheritance] = {"Onset": {'base': [onset_min], 'x': [onset_max-onset_min], 'y': [disease], 'color': [inheritance]}, 
+                                            "TypOnset": {'base': [typonset_min], 'x': [typonset_max-typonset_min], 'y': [disease], 'color': inheritance}}
+            else:
+                if x in plot_data:
+                    plot_data[x]["Onset"]["y"].append(disease)
+                    plot_data[x]["Onset"]["base"].append(0)
+                    plot_data[x]["Onset"]["x"].append(0)
+                    plot_data[x]["TypOnset"]["y"].append(disease)
+                    plot_data[x]["TypOnset"]["base"].append(0)
+                    plot_data[x]["TypOnset"]["x"].append(0)
+                else:
+                    plot_data[x] = {"Onset": {'base': [0], 'x': [0], 'y': [disease]}, 
+                                    "TypOnset": {'base': [0], 'x': [0], 'y': [disease]}}
+
+    fig = go.Figure(layout=go.Layout()) #width = 600, height=len(yvalues_line)*10)
+
+    fig.update_layout( yaxis=dict( tickmode="array", tickvals=diseases))
+
+    for key in plot_data:
+        fig.add_trace(go.Bar(
+            y=plot_data[key]['Onset']['y'],
+            x=plot_data[key]['Onset']['x'],
+            base=plot_data[key]['Onset']['base'],
+            name= key,
+            legendgroup=key,
+            orientation = 'h',
+            marker=dict(color=colors[key]),
+            width=0.2,
+            offset=-0.1,
+            hovertemplate="Disease: %{y} <br> Range onset: %{base} - %{x} years"
+        ))
+    for key in plot_data:
+        fig.add_trace(go.Bar(
+            y=plot_data[key]['TypOnset']['y'],
+            x=plot_data[key]['TypOnset']['x'],
+            base=plot_data[key]['TypOnset']['base'],
+            name= key,
+            legendgroup=key,
+            orientation = 'h',
+            marker=dict(color=colors[key]),
+            width=0.6,
+            offset=-0.3,
+            hovertemplate="Disease: %{y} <br> Typical onset: %{base} - %{x} years",
+            showlegend=False
+        ))
+
+    fig.add_trace(go.Scatter(
+        y=[diseases[0], diseases[-1]],
+        x=[18, 18],
+        mode="lines",
+        line=dict(dash='dot', color='#aeaeae'),
+        showlegend=False,
+        hoverinfo='text',
+        hovertemplate="Age 18",
+        name="Age 18"
+    ))
+
+    fig.update_layout(autosize=False, 
```

Comment:
I wonder if this line might be the culprit. In javascript when I'm using the Plotly library, I'm passing a seemingly similar option `responsive: true`, which was working before. But maybe setting this param here overrides it.